cibuildwheel/__main__.py

```diff
@@ -107,10 +107,8 @@ def main():
 
     try:
```

Comment:
I believe this whole try-except block can go. The only reason for having it was to get the project's name, but since we're removing it, I don't believe it's still necessary. And it will probably solve the build failure on the CI services.